references/cli/status-ui/model/info.go

```diff
@@ -0,0 +1,67 @@
+package model
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/oam-dev/kubevela/apis/types"
+	"github.com/oam-dev/kubevela/pkg/utils/helm"
+	"github.com/oam-dev/kubevela/version"
+)
+
+type Info struct {
+	cluster string
+}
+
+const (
+	UNKOWN_VERSION = "UNKNOWN"
+)
+
+func NewInfo() *Info {
+	return &Info{
+		cluster: "local",
+	}
+}
+
+func (i Info) Cluster() string {
+	return i.cluster
+}
+
+func (i Info) K8SVersion(config *rest.Config) string {
```

Comment:
I suggest to move it to utils, because it is common function, which can be reused.

Reply:
Sorry, I think this function is appropriate here.

Reply:
ok, I have fixed it.

Reply:
Because these functions are used to load system information, I put them in `info.go` here, some functions in this file depend on the `Info` structure, so it is not suitable to move them out, so it is better to put them together